src/app/views/collections/validation/collectionValidation.js

```diff
@@ -26,7 +26,10 @@ export default class collectionValidation {
             };
         }
 
-        if (collections && (collections.some(c => c.name === name.trim()) || collections.some(c => c.id.split("-")[0] === name.replace(/\s+/g, "")))) {
+        name = name.trim()
+        name = name.replace(/\s+/g, "")
```

Comment:
Wouldn't the `replace` function make the `trim` unnecessary? 
Also, I'd suggest using a new variable instead of reusing `name` - it is harmless _at the moment_ as we don't use `name` after this but it is good practice not to reassign function parameters

Reply:
Actually, no. If we want to keep the current behaviour, we need both replace and trim, but in different variables. We check the name against a trimmed version of the current name but the id against the whitespace-free version of the name. If we replace all the spaces, the name check would fail. Do the unit tests pass with this code?

Reply:
They do.